src/App.jsx

```diff
@@ -33,29 +37,41 @@ function App() {
   const onStart = () => {
     setStart(false);
     setReset(false);
+    setShowResult(false);
   };
 
   const onReset = () => {
     setReset(true);
     setStep(0);
     setCorrectAnswers(0);
     setStart(true);
+    setShowResult(false);
   };
 
   useEffect(() => {
     const fetchData = async () => {
       try {
-        const res = await axios.get(API_URL);
+        const userToken = await axios.get(TOKEN);
+        const { token, response_code } = userToken.data;
+        setToken(token);
+        setResCode(response_code);
```

Comment:
Can you find a way to simplify the code by removing `setResCode`?

Reply:
I tried via variables but not sure